app/controllers/papers_controller.rb

```diff
@@ -16,6 +16,7 @@ def show
   # GET /papers/new
   def new
     @paper = Paper.new
+    @authors = 5.times {@paper.authors.build}
```

Comment:
Is there a reason to split it into 2 lines?  I just want to make sure I understand the logic for future reference.  Easier to read?